### DISCUSSION TITLE
Calendar icon left-positioned

### DISCUSSION THREAD

**Discussion by joaomarcelofm** [primefaces/community#1498] [PrimeVue]
The current `Calendar` icon props inject the calendar icon on the right side of the component. However, it differs from the new `IconField` position. This makes the UI look strange when these icons are used together, as if there's no consistency across icon positions.

### Suggestion
Add prop to position the icon on either sides.

### Pull request
https://github.com/primefaces/primevue/pull/5460

**Comment by joaomarcelofm** [comment]
@tugcekucukoglu, thanks for your feedback on the PR above!

Could you please let me know if this feature is in the roadmap though? I see it's available in [PrimeReact](https://primereact.org/calendar/#api.Calendar.props.iconPos) but not in PrimeVue, and the inability to do this means that we have to create a wrapper component around the Calendar component to achieve such behavior.

**Comment by hartum** [comment]
My solution:
**In template:**

` <DatePicker
            v-model="date"
            showIcon
            iconDisplay="input"
            inputId="icondisplay"
            selectionMode="range"
            hideOnRangeSelection
            :manualInput="false"
            :numberOfMonths="2"
            class="custom-datepicker"
        />`

**In style**
`.custom-datepicker {
    .p-datepicker-input {
        padding-inline-start: calc(
            (var(--p-form-field-padding-x) * 2) + var(--p-icon-size)
        );
    }
    .p-datepicker-input-icon-container {
        inset-inline-start: var(--p-form-field-padding-x);
        inset-inline-end: unset;
    }
}`
